setup.sh

```diff
@@ -127,22 +124,50 @@ function install_vagrant_plugins {
   vagrant_plugin_install vagrant-gatling-rsync "Vagrant Gatling Rsync"
 }
 
+# env_is_defined VARNAME
+#   Checks if a new $SHELL has $VARNAME defined in its environment.
+#   Returns 0 when VARNAME is defined for new shells, 1 otherwise.
+function env_is_defined {
+  local var="$1"
+  if test -z "${var}" ; then
```

Comment:
Done. I also added a tiny backtrace() function which I've found useful in other bash scripts.